src/core/tokenStore.js

```diff
@@ -204,56 +194,48 @@ export default function tokenStoreFactory(options) {
          * (Could happen if maxSize is reduced during the life of the tokenStore)
          * @returns {Promise} - resolves when done
          */
-        enforceMaxSize: function enforceMaxSize() {
-            var self = this;
-            return this.getIndex().then(function(latestIndex) {
-                var keysToRemove;
-                var excess = latestIndex.length - config.maxSize;
-                if (excess > 0) {
-                    keysToRemove = latestIndex.slice(0, excess);
-                    return Promise.all(
-                        _.map(keysToRemove, function(key) {
-                            return self.remove(key);
-                        })
-                    );
-                }
-                return true;
-            });
+        enforceMaxSize() {
+            return this.getIndex()
+                .then(latestIndex => {
+                    const excess = latestIndex.length - config.maxSize;
+                    if (excess > 0) {
+                        const keysToRemove = latestIndex.slice(0, excess);
+                        return Promise.all(
+                            _.map(keysToRemove, key => this.remove(key))
```

Comment:
Could be `keysToRemove.map(key => this.remove(key))`

Reply:
Indeed, no need to use lodash here :)